test/Interop/CxxToSwiftToCxx/bridge-cxx-struct-back-to-cxx.swift

```diff
@@ -21,8 +21,6 @@
 // RUN: %target-interop-build-clangxx -fsyntax-only -x c++-header %t/full-cxx-swift-cxx-bridging.h -std=gnu++17 -c -fmodules -fcxx-modules -I %t 
 // FIXME: test c++20 (rdar://117419434)
 
-// XFAIL: OS=linux-android, OS=linux-androideabi
```

Comment:
Is this backwards compatible?

Reply:
With NDK 26? I think this was fixed by the new Android overlay, so it should be.